### ISSUE TITLE
Visually disable components that have aria-disabled?

### ISSUE THREAD

**Issue by oddvernes** [equinor/design-system#2544]
In some cases, users would want to use `aria-disabled` instead of `disabled` on buttons and other disableable components, which ensures the elements is still keyboard-navigatable. But we do not apply the disabled style in these cases, which we should to make this usable.

**Comment by oddvernes** [comment]
So the replacement should be `&:is(:disabled, [aria-disabled]){...}` instead of `&:disabled{...}`
